modules/imgproc/src/colormap.cpp

```diff
@@ -499,11 +498,8 @@ namespace colormap
             CV_Error(Error::StsAssert, "cv::LUT only supports tables of size 256.");
         Mat src = _src.getMat();
         // Return original matrix if wrong type is given (is fail loud better here?)
```

Comment:
done. There is also comment in brief docs, but I am waiting for PR 6150 to be accepted (otherwise there will be conflict).